docs/Antora/modules/getting_started/pages/quick_start_setting_up_csharp.adoc

```diff
@@ -0,0 +1,113 @@
+= Setting up (C#)
+
+To start working with openDAQ(TM), the requisite binaries are required. They can be obtained at {docs-website}[the openDAQ(TM) documentation and releases webpage] or from http://nuget.org (recommended way). 
+
+NOTE: The .NET binaries are currently only available in the Windows(TM) SDK packages. For Linux(TM) they are included in the NuGet package, although here the .NET Bindings are not tested.  
+
+All commands in this guide use the .NET command-line interface (dotnet CLI) which is a cross-platform toolchain for developing, building, running, and publishing .NET applications.
+
+== Creating an openDAQ(TM) project
+
+Below, we start a Console Project in Visual Studio(TM) from scratch but there are also example files which can be found in the examples archive from the {docs-website}[openDAQ(TM) documentation and releases webpage].
+
+=== Requirements
+
+[tabs]
+====
+Windows::
++
+--
+ * .NET compiler, e.g.
+   - https://learn.microsoft.com/en-us/dotnet/core/sdk[.NET SDK] (used in this guide).
+   - Visual Studio 2022(TM).
+   - Visual Studio Code with extension "C# Dev Kit".
+--
+
+Linux::
++
+--
+ * Ubuntu 22.04 (see also the note below).
+ * On Windows(TM) users can just cross-compile the project for Linux(TM) using the Windows(TM)-requirements (used in this guide).
+ * For development (not tested) and running on Linux(TM): +
+   https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-install?tabs=dotnet8&pivots=os-linux-ubuntu-2204[Install .NET SDK or .NET Runtime on Ubuntu].
+
+NOTE: On Windows there is the https://learn.microsoft.com/en-us/windows/wsl/install[Windows Subsystem for Linux (WSL)] where the cross-compiled executables can be tested.
+--
+====
+
+In general everything around .NET can be found here: +
+https://dotnet.microsoft.com/en-us/download/dotnet/8.0
+
+=== Creating and building the project
+
+In this guide we use a `Developer PowerShell for VS2022` prompt on Windows(TM) for creating and (cross-) building the project. +
+For the cross-compilation we use a `bash` prompt on Linux(TM) (or WSL on Windows(TM)) to run the result.
+
+To start, create a `csharp/quick_start` directory and navigate to it.
+
+With  the following commands we first create an empty (.NET8.0) console application project and then we add the _openDAQ.Net_ Bindings NuGet package reference to the project, which contains all the needed binaries.
+
+[source,shell]
+----
+dotnet new console
+dotnet add package openDAQ.Net
```

Comment:
This command fails for me with:
```
info : X.509 certificate chain validation will use the default trust store selected by .NET for code signing.
info : X.509 certificate chain validation will use the default trust store selected by .NET for timestamping.
info : Adding PackageReference for package 'openDAQ.Net' into project 'C:\Users\aljaz\csharp\csharp.csproj'.
info :   GET https://api.nuget.org/v3/registration5-gz-semver2/opendaq.net/index.json
info :   NotFound https://api.nuget.org/v3/registration5-gz-semver2/opendaq.net/index.json 129ms
info :   GET https://api.nuget.org/v3/registration5-gz-semver2/opendaq.net/index.json
info :   NotFound https://api.nuget.org/v3/registration5-gz-semver2/opendaq.net/index.json 193ms
error: There are no versions available for the package 'openDAQ.Net'.
```

Reply:
The command you provided gives:
```
Option '--source' expects a single argument but 2 were provided.
```
However, this worked:
```
dotnet add package openDAQ.Net --source path/to/folder
```

I would write the guide as follows:
With the following command we first create an empty (.NET8.0) console application project:
```
dotnet new console
```
Then we add the openDAQ.Net Bindings NuGet package reference to the project, which contains all the needed binaries:
```
dotnet add package openDAQ.Net
```
Alternatively, the latest versions of the C# bindings are available at https://docs-dev.opendaq.com/ and can be used via:
```
dotnet add package openDAQ.Net --source path/to/folder
```